workbench/soapclient/SforceBaseClient.php

```diff
@@ -94,9 +94,10 @@ public function printDebugInfo() {
     /**
      * Connect method to www.salesforce.com
      *
-     * @param string $wsdl   Salesforce.com Partner WSDL
+     * @param string  $wsdl                        Salesforce.com Partner WSDL
+     * @param ConnectionConfiguration $connConfig  Connection config
      */
-    public function createConnection($wsdl, $proxy=null) {
+    public function createConnection($wsdl, ConnectionConfiguration $connConfig, $proxy=null) {
```

Comment:
don't know whether it is safe to insert parameter like this but since $proxy not used anywhere I've assumed I can hijack parameter #2 and someone can pass proxy by name if they want to